Papr/Utils/Extentions/Date+Extentions.swift

```diff
@@ -40,7 +40,9 @@ extension Date {
         } else if roundedDate >= 24 * 60.0 && roundedDate < 7 * 24 * 60 {
             return "\(Int(self.since(Date(), in: .day).rounded()))d"
         } else if roundedDate >= 7 * 24 * 60.0 && roundedDate < 30 * 24 * 60 {
-            return "\(Int(self.since(Date(), in: .week).rounded()))w"
+            let dateformatter = DateFormatter()
```

Comment:
Issue #8 suggests to show the full date if the photo date is greater than 1 week. In here you are handling the case only between 1 week and 1 month.

Reply:
Also the date format should be `dd/mm/yy`

Reply:
sorry i will change that

Reply:
hey i removed the condition `roundedDate < 30 * 24 * 60` what about other case is it required